coding-guides/cleverbot-integration.md

```diff
@@ -30,7 +30,7 @@ client.login("SuperSecretBotTokenHere");
 Once you've got that, we should go check out `cleverbot-node` on [npmjs.com](https://www.npmjs.com/package/cleverbot-node) and grab their example code
 
 ```javascript
-var Cleverbot = require("cleverbot-node");
+let Cleverbot = require("cleverbot-node");
```

Comment:
Kinda needless change, as the next time this code is shown it's updated.

Reply:
It's better to use `let` than `var`

Reply:
I'll change it back to `var` in the next commit for consistency

Reply:
Changed it back to `var` :ok: